Writerside/topics/Sessions.md

```diff
@@ -0,0 +1,129 @@
+# Sessions
+
+Sessions are typically used to represent users after a successful login.
```

Comment:
The first paragraph is used when we link to this page elsewhere in the popup, could we add some more detail about what it contains/ simplest use cases